packages/opencensus-instrumentation-ioredis/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@opencensus/instrumentation-ioredis",
+  "version": "0.0.1",
+  "description": "Opencensus IORedis automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "census-instrumentation/opencensus-node",
+  "scripts": {
+    "test": "nyc mocha build/test/**/*.js",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json",
+    "clean": "rimraf build/*",
+    "check": "gts check",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check"
+  },
+  "keywords": [
+    "opencensus",
+    "nodejs",
+    "tracing",
+    "profiling"
+  ],
+  "author": "Google Inc.",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "CHANGELOG.md",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@types/mocha": "^5.2.6",
+    "@types/nock": "^9.1.3",
+    "@types/node": "^10.12.12",
+    "@types/semver": "^5.5.0",
+    "@types/shimmer": "^1.0.1",
+    "@types/uuid": "^3.4.3",
+    "codecov": "^3.1.0",
+    "gts": "^0.9.0",
+    "mocha": "^6.0.0",
+    "ncp": "^2.0.0",
+    "nock": "^10.0.0",
+    "nyc": "^13.0.0",
+    "rimraf": "^2.6.2",
+    "ts-node": "^8.0.0",
+    "typescript": "~3.2.0",
+    "@types/ioredis": "^4.0.9",
```

Comment:
Can you move @types on top, we try to list dependencies in sorted order. Also looks like @types/uuid is not required here.